openff/toolkit/topology/molecule.py

```diff
@@ -1255,17 +1256,27 @@ def __hash__(self):
         return hash(self.to_smiles())
 
     def ordered_connection_table_hash(self) -> int:
-        """Compute an ordered hash of the atoms and bonds in the molecule"""
+        """
+        Compute an ordered hash of the atoms and bonds in the molecule.
+        This hash method is intended for comparison of Molecule objects at
+        runtime, and hashes from one version of the software should not be
+        compared with hashes generated using different versions.
+        """
```

Comment:
```suggestion
        """
        Compute an ordered hash of the atoms and bonds in the molecule.
        
        This hash method is intended for comparison of Molecule objects at
        runtime, and hashes from one version of the software should not be
        compared with hashes generated using different versions.
        """
```
Sorry for the pedantry.